tools_helpers/check_reasoning_models.py

```
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+"""
+Helper script to test which OpenAI models support reasoning through LiteLLM.
+
+This script tests various OpenAI models to determine which ones support
+reasoning capabilities for the NEAR Catalyst Framework analysis workflow.
+"""
+
+import litellm
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)